csrc/ir/internal_nodes.h

```diff
@@ -2609,9 +2614,14 @@ output = [N, H, L, Ev]
 logsumexp = [N, H, L]
 dropout_p = scalar(double)
 is_causal = scalar(bool)
-philox_seed = scalar CPU tensor
-philox_offset = scalar CPU tensor
-scale = scalar(double)
+philox_seed = CPU scalar tensor or uint64_t[2] tensor (for > 2.7.0)
+philox_offset = CPU scalar tensor or empty uint64_t tensor (for > 2.7.0)scale =
+scalar(double)
+
+Note: For older versions, torch returns CPU scalar tensors for philox_seed and
+philox_offset. For torch 2.7.0 and above, torch returns philox_seed -> rng_state
+(uint64_t[2]) and philox_offset -> _unused (empty tensor). The rng state
+contains both seed and offset.
```

Comment:
nitpick: I totally get what you meant, but the comment here for SdpaBwd, philox_seed & philox_offset are inputs, instead of outputs. :shrug: 

FYI, might be easier to just refer to the comment on line 2288 - 2291, instead of duplicate it here.